vendor/github.com/pkg/errors/errors.go

```diff
@@ -1,310 +0,0 @@
-//     case *MyError:
-//             // handle specifically
-//     default:
-//             // unknown error
-//     }
-//
-// causer interface is not exported by this package, but is considered a part
-// of stable public API.
-//
-// Formatted printing of errors
-//
-// All error values returned from this package implement fmt.Formatter and can
-// be formatted by the fmt package. The following verbs are supported
-//
-//     %s    print the error. If the error has a Cause it will be
-//           printed recursively
-//     %v    if Debug(true) has not been called, this is equivalent to %s.
-//           Otherwise it is equivalent to %+v.
-//     %+v   extended format. Each Frame of the error's StackTrace will
-//           be printed in detail.
-//
-// Retrieving the stack trace of an error or wrapper
-//
-// New, Errorf, Wrap, and Wrapf record a stack trace at the point they are
-// invoked. This information can be retrieved with the following interface.
-//
-//     type stackTracer interface {
-//             StackTrace() errors.StackTrace
-//     }
-//
-// Where errors.StackTrace is defined as
-//
-//     type StackTrace []Frame
-//
-// The Frame type represents a call site in the stack trace. Frame supports
-// the fmt.Formatter interface that can be used for printing information about
-// the stack trace of this error. For example:
-//
-//     if err, ok := err.(stackTracer); ok {
-//             for _, f := range err.StackTrace() {
-//                     fmt.Printf("%+s:%d", f)
-//             }
-//     }
-//
-// stackTracer interface is not exported by this package, but is considered a part
-// of stable public API.
-//
-// See the documentation for Frame.Format for more details.
-package errors
-
-import (
-	"fmt"
-	"io"
-	"sync"
-)
-
-// boolMutex combines a sync.RWMutex with a bool where each method is correctly
-// handled by locking and unlocking the mutex appropriately so you can use it
-// in one-liners.
-type boolMutex struct {
-	m     *sync.RWMutex
-	value bool
-}
-
-// debugEnabled defines whether debugging is currently globally enabled. If it
-// is enabled then Format (with a formatting flag of "%v") should always return
-// a stack-based output (even if "+" is not set).
-var debugEnabled = &boolMutex{
-	m:     new(sync.RWMutex),
-	value: false,
-}
-
-// Set sets the value stored in bm to the provided boolean.
-func (bm *boolMutex) Set(new bool) {
-	bm.m.Lock()
-	bm.value = new
-	bm.m.Unlock()
-}
-
-// Get returns the current value stored in bm.
-func (bm *boolMutex) Get() bool {
-	bm.m.RLock()
-	value := bm.value
-	bm.m.RUnlock()
-	return value
-}
-
-// Debug sets whether or not debugging is enabled. If enabled, then formatting
-// an error wrapped by pkg/error with "%v" will return a full stack trace. This
-// allows you to dynamically decide whether to output stack traces (without
-// having to use "%+v" indiscriminately).
-func Debug(value bool) {
-	debugEnabled.Set(value)
-}
```

Comment:
Not sure if you noticed this, but this is actually a patched version of `pkg/error`. Though we don't *really* need this anymore, so no worries.

Reply:
Yes, I noticed; I did the first commit separately for that;

> This commit is mostly to preserve history of the local changes that were made to the pkg/errors package (which appear to have been made for debugging purposes). (...)

Reply:
Sorry, my fault for not reading :bow:.

Reply:
No worries! It was a good catch 👍